src/github.com/matrix-org/go-neb/database/schema.go

```diff
@@ -45,12 +45,14 @@ CREATE TABLE IF NOT EXISTS auth_realms (
 );
 
 CREATE TABLE IF NOT EXISTS auth_sessions (
+	id TEXT NOT NULL,
```

Comment:
Can we call it session_id? It makes it clearer whats going on when constructing silly SQL.